be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -495,17 +519,11 @@ ExchangeSinkOperatorFactory::ExchangeSinkOperatorFactory(int32_t id, int32_t pla
           _partition_expr_ctxs(std::move(partition_expr_ctxs)) {}
 
 OperatorPtr ExchangeSinkOperatorFactory::create(int32_t degree_of_parallelism, int32_t driver_sequence) {
-    if (_part_type == TPartitionType::UNPARTITIONED || _destinations.size() == 1) {
-        return std::make_shared<ExchangeSinkOperator>(_id, _plan_node_id, _buffer, _part_type, _destinations,
-                                                      _sender_id, _dest_node_id, _partition_expr_ctxs);
-    } else if (_part_type == TPartitionType::HASH_PARTITIONED ||
-               _part_type == TPartitionType::BUCKET_SHFFULE_HASH_PARTITIONED) {
-        return std::make_shared<ExchangeSinkOperator>(_id, _plan_node_id, _buffer, _part_type, _destinations,
-                                                      _sender_id, _dest_node_id, _partition_expr_ctxs);
-    } else {
-        DCHECK(false) << " Shouldn't reach here!";
-        return nullptr;
-    }
+    DCHECK(_part_type == TPartitionType::UNPARTITIONED || _destinations.size() == 1 ||
```

Comment:
Could remove this DCHECK, There is no meaning. Before there is  no random partition type in BE, currently, BE could have all partition types

Reply:
@kangkaisen There is an `RANGE_PARTITIONED` partition type we doesn't consider. Is `RANGE_PARTITIONED` deprecated?